### DISCUSSION TITLE
Status of EFI ISO image

### DISCUSSION THREAD

**Discussion by tlschenkjr** [PartialVolume/shredos.x86_64#250] [General]
The README.md contains the following statement:

"**The .iso image currently supports legacy bios booting only and not UEFI, however, a bios/UEFI version of the .iso is in development and will be released shortly.**"

Is there any status update on the bios/EFI version of the ISO?  I'm currently stuck building the custom images for our environment due to the lack of an EFI iso image.  We recently upgraded to a new hardware platform and it doesn't support legacy BIOS mode, and inserting a USB stick into literally thousands of servers is not feasible.  We are also investigating PXE booting Shredos, so that we could use our Satellite server to run Shredos on multiple servers, but it would be nice to have the ISO option as well.

Any information would be greatly appreciated and if you need someone to test a bios/EFI image, I would be happy to assist.

Thanks,

Tom S.

**Comment by sorenstoutner** [comment]
I also would be interested in knowing the expected timeline for EFI support in the ISO.
